libs/apnos/apnos.py

```diff
@@ -314,43 +314,44 @@ def run_generic_command(self, cmd=""):
         return status
 
     def get_ucentral_status(self):
+        connected, latest, active = "Error", "Error", "Error"
         try:
             client = self.ssh_cli_connect()
             cmd = "ubus call ucentral status"
             if self.mode:
                 cmd = f"cd ~/cicd-git/ && ./openwrt_ctl.py {self.owrt_args} -t {self.tty} --action " \
                       f"cmd --value \"{cmd}\" "
             stdin, stdout, stderr = client.exec_command(cmd)
-            output = stdout.read()
-            # print(output, stderr.read())
-            connected = False
-            if "connected" in output.decode('utf-8').splitlines()[2]:
+            output = stdout.read().decode("utf-8")
+            output_formatted = ''.join(output.split())
+            json_output = json.loads(output_formatted)
```

Comment:
@tusharmetkar this will not work here because the output_formatted also contains command.
I have done a workaround and did a split and then collected it. i have done a fix for this on library level in master branch. please check and rebase

Reply:
@shivamcandela  I've rebased the branch with master and printed the out and it looks good.
{code}
output == {
	"connected": 1245,
	"latest": 1632162537,
	"active": 1632162537
}

output_formatted == {"connected":1245,"latest":1632162537,"active":1632162537}
{code}

Reply:
@tusharmetkar this is not working in TIP Labs
Are u using ssh or serial console?

Reply:
:~# ubus call ucentral status
{
	"connected": 65,
	"latest": 1632384879,
	"active": 1632384879
}
This is output, which can't be converted into json, that's why it is failing